pennylane/circuit_graph.py

```diff
@@ -469,9 +471,57 @@ def get_depth(self):
                 self._operation_graph = self._graph.subgraph(
                     list(self._graph.nodes().index(node) for node in self.operations)
                 )
-                self._depth = rx.dag_longest_path_length(self._operation_graph) + 1
+                self._extend_graph(self._operation_graph)
```

Comment:
The is still called even if there is not custom wire with depth > 1 ?

Reply:
Yes! If there are no custom operations with depth > 1 then the `extend_graph` method will do nothing to the graph.